flink-streaming-java/pom.xml

```diff
@@ -108,6 +108,7 @@ under the License.
 				<artifactId>maven-surefire-plugin</artifactId>
 				<configuration>
 					<reuseForks>false</reuseForks>
+					<argLine>-XX:-UseSplitVerifier</argLine>
```

Comment:
there is currently a problem with the tests that i assume are caused by powermock. i found this as a solution to the issue, although other problems still remain with the tests atm...